.github/workflows/ci.yml

```diff
@@ -0,0 +1,37 @@
+name: Python CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+  run_tests:
+    name: Tests
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+          - ubuntu-20.04
+        python-version:
+          - 3.8
+        toxenv: [ testenv ]
```

Comment:
Why does this field use `[]` syntax while the others use `-` syntax?  It will be easier to read if they are the same.

Reply:
Updated all the fields to use `[]` notation.